src/pages/Signup/index.js

```diff
@@ -47,80 +48,119 @@ const Signup = () => {
+    to: '',
+  });
+
+  const sendEmailValidation = data => {
+    setToSend(prevState => {
+      return {
+        ...prevState,
+        to_name: data.username,
+        to: data.email,
+      };
+    });
+  };
+
+  const onSubmit = data => {
+    sendEmailValidation(data);
+    signup(data);
+  };
 
   const resetErrors = useCallback(() => dispatch(api.util.resetApiState()), [dispatch]);
 
   const handleFocus = () => error && resetErrors();
 
-  useEffect(() => {
-    if (isSuccess) {
-      push(routesPaths.index);
-    }
-  }, [isSuccess, user, push]);
+  useEffect(() => {}, [isSuccess, user, push]);
 
   useEffect(() => resetErrors, [resetErrors]);
-
-  if (authenticated) {
-    return <Redirect to={routesPaths.index} />;
-  }
+  useEffect(() => {
+    if (!isSend) {
+      emailjs
+        .send('service_c2myo6u', 'template_zdq8t9e', toSend, 'CzOOH1L12ZmTTkiFU')
+        .then(response => {
+          setIsSend(false);
+        })
+        .catch(err => {
+          console.log('FAILED...', err);
+        });
+    }
+  }, [toSend, isSend]);
 
   return (
     <div className="row">
-      <div className="form column left-column">
-        <form onSubmit={handleSubmit(onSubmit)} noValidate>
-          <h1>{t('signup.title')}</h1>
-          <label htmlFor="username">Name</label>
-          <Input
-            register={register}
-            type="text"
-            name="username"
-            error={errors.username}
-            handleFocus={handleFocus}
-          />
-          <label htmlFor="email">{t('signup.labels.email')}</label>
-          <Input
-            register={register}
-            type="email"
-            name="email"
-            error={errors.email}
-            handleFocus={handleFocus}
-          />
-
-          <label htmlFor="password">{t('signup.labels.password')}</label>
-          <Input
-            register={register}
-            type="password"
-            name="password"
-            error={errors.password}
-            handleFocus={handleFocus}
-          />
-
-          <label htmlFor="password">{t('signup.labels.passwordConfirmation')}</label>
-          <Input
-            register={register}
-            type="password"
-            name="passwordConfirmation"
-            error={errors.passwordConfirmation}
-            handleFocus={handleFocus}
-          />
-          <label htmlFor="gender">Gender</label>
-          <ComboBox
-            register={register}
-            name="gender"
-            error={errors.gender}
-            handleFocus={handleFocus}
-            dataSource={genderList}
-          />
-          <div className="button-container">
-            <Button type="submit" disabled={isLoading}>
-              {t('signup.title')}
-            </Button>
-            <Link to={routesPaths.login}>{t('signup.alreadyHaveAccount')}</Link>
-          </div>
-        </form>
-      </div>
+      {isSend === true ? (
```

Comment:
It's not necessary to declare this conditional as `isSend === true`, if you write `isSend ?` it would true or false either way.